_posts/2016-03-07-2016-forman-survey-analysis.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: post
+title: 2016 Foreman Survey Analysis
+date: 2016-03-07 14:34:44
+author: Greg Sutcliffe
+tags:
+- usage
+- foreman
+- events
+- plugins
+- community
+- survey
+- API
+- virtualization
+- cfgmgmtcamp
+- puppet
+- devconf
+- fosdem
+- chef
+- use cases
+---
+
+Over the course of Febrary, we again ran the Foreman Community Survey to get
+feedback on how we're doing and what needs to be addressed. I've now had some
+time to sit down and look through the data, and share some insights with you
+all.
+
+Firstly - **thank you** to all those who filled out the survey. It was a little
+bit longer this year, as we looked to get data on how we can grow the
+contributor community as well as the user community, so thanks for taking the
+time. It's great to hear all your feedback.
+
+<!--more-->
+
+## <a name="intro"></a>Contents
+
+* [Intro](/2016/03/2016-forman-survey-analysis.html/#intro)
+* [Community Diversity](/2016/03/2016-forman-survey-analysis.html/#diversity)
+* [Support and Docs](/2016/03/2016-forman-survey-analysis.html/#support)
+* [Specific Features](/2016/03/2016-forman-survey-analysis.html/#foreman-features)
+* [Upcoming Work](/2016/03/2016-forman-survey-analysis.html/#upcoming-work)
+* [Events](/2016/03/2016-forman-survey-analysis.html/#events)
+* [Contributor Survey](/2016/03/2016-forman-survey-analysis.html/#contributor-survey)
+* [Final thoughts](/2016/03/2016-forman-survey-analysis.html/#final-thoughts)
+
+Sadly, response were slightly down from last year (**76 responses**) and the
+smaller sample size makes it difficult to draw strong conculsions. According to
+other metrics, we have usually 250ish active people in IRC each month, and some
+2000 people on the mailing list. Suggestions on how to reach more people with
+the survey are, of course, [welcome][contact]!
+
+That said, we can still look at the results and look for trends, especially as
+compared to last year (which we couldn't do then, it being the first such
+survey). Lets start with some stats on...
+
+## <a name="diversity"></a>Community Diversity
```

Comment:
They're anchors for the table of contents at the top - markdown doesn't seem to be generating them for normal headers.